Dockerfile

```diff
@@ -1,4 +1,23 @@
 FROM bitnami/spark:3.5.1
 
+RUN export ORI_USER=$(id -u)
+# Switch to root to install packages
+USER root
 
-ENTRYPOINT ["sleep 10s"]
+ENV PYTHON_VER=python3.11
+
+# Install necessary packages
+RUN apt-get update && apt-get install -y \
+    $PYTHON_VER python3-pip $PYTHON_VER-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install Jupyterlab and other python dependencies
+RUN pip3 install jupyterlab==4.2.0 pyspark==3.5.1
+
+COPY scripts/entrypoint.sh /opt/
+RUN chmod a+x /opt/entrypoint.sh
+
+# Switch back to the original user
+#USER ${ORI_USER}
```

Comment:
is this supposed to be commented out now?

Reply:
oh oh. I was testing things and did `add .`

Reply:
:+1: